apps/lend/src/hooks/useVaultShares.ts

```diff
@@ -24,7 +24,7 @@ function useVaultShares(rChainId: ChainId, rOwmId: string, vaultShares: string |
       const borrowedAmtUsd = +pricePerShare * +vaultShares * +usdRate
 
       return {
-        borrowedAmount: `${formatNumber(borrowedAmt, { showDecimalIfSmallNumberOnly: true })} ${symbol}`,
+        borrowedAmount: `${formatNumber(borrowedAmt, { minimumFractionDigits: 2, maximumFractionDigits: 2 })} ${symbol}`,
```

Comment:
```suggestion
        borrowedAmount: `${formatNumber(borrowedAmt, { minimumSignificantDigits: 6, maximumSignificantDigits: 6 })} ${symbol}`,
```

Reply:
With it being non-perfect, is it 'good enough' or are we in need for a better solution? If it's 'good enough' can you quickly add a comment explaining this edge case where it's behaving slightly different than expected?

Reply:
I was talking about the suggested change, not the committed one. I think the committed change is good enough for now.
We can improve it further later